xerosdk/apis/bank_transactions.py

```diff
@@ -22,7 +22,7 @@ def get_all(self):
             List of all Bank Transactions
         """
 
-        return self._get_request(BankTransactions.GET_BANK_TRANSACTIONS)
+        return list(self._get_all(BankTransactions.GET_BANK_TRANSACTIONS, 'BankTransactions'))
```

Comment:
are we going to default the generator approach for apis? Shouldn't generator be an additional option?

Reply:
we can discuss this in tomorrow sync up